custom-commands/demos/hospitality/skill/en-us/HospitalityDemo.json

```diff
@@ -116,15 +125,209 @@
+      "- How much do you want to change the temperature by?",
+      "- By how many degrees?"
+    ],
+    "# AdjustTemperatureRoomControlBackendonSuccess64368a3028e111eaaf158b43807e75e2": [
+      "- Type your response(s) here."
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailure64368a3028e111eaaf158b43807e75e2": [
+      "- Sorry, not able to connect to the room control service right now"
+    ],
+    "# AdjustTemperatureRoomControlBackendonSuccess958d9ce028e111eaaf158b43807e75e2": [
+      "- Type your response(s) here."
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailure958d9ce028e111eaaf158b43807e75e2": [
+      "- Sorry, unable to connect to the room control service right now"
+    ],
+    "# AdjustTemperatureRoomControlBackendonSuccessd002848028e111eaaf158b43807e75e2": [
+      "- Type your response(s) here."
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailured002848028e111eaaf158b43807e75e2": [
+      "- Sorry, unable to connect to the room control service right now"
+    ],
+    "# AdjustTemperature2c2602f028e211eaaf158b43807e75e2": [
+      "- Do you want to make it warmer or cooler?"
+    ],
+    "# AdjustTemperature571bc8a028e211eaaf158b43807e75e2": [
+      "- What temperature do you want to set it to?"
+    ],
+    "# AdjustTemperatureRoomControlBackendonSuccess287895e028e311ea8db34fd5c8a135ab": [
+      "- {RoomControlBackend.Message}"
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailure287895e028e311ea8db34fd5c8a135ab": [
+      "- Sorry, not able to connect to the room control service right now"
+    ],
+    "# AdjustTemperatureRoomControlBackendonSuccess43dd866028e311ea8db34fd5c8a135ab": [
+      "- {RoomControlBackend.Message}"
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailure43dd866028e311ea8db34fd5c8a135ab": [
+      "- Sorry, not able to connect to the room control service right now"
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailure76ee7e1028e311ea8db34fd5c8a135ab": [
+      "- Sorry, unable to connect to the room control service right now"
+    ],
+    "# AdjustTemperature4519903028e011ea83cf95b9c7d5480f": [
+      "- I can only set between 68 and 80 degrees.  What temperature would you like?"
+    ],
+    "# AdjustTemperatureRoomControlBackendonSuccessfd34ee9028f311ea88cde3d2ef24c102": [
+      "- {RoomControlBackend.Message}"
+    ],
+    "# AdjustTemperatureRoomControlBackendonFailurefd34ee9028f311ea88cde3d2ef24c102": [
+      "- Sorry, unable to connect to the room control service right now"
+    ],
+    "# OpenClose-RoomControlBackendonSuccess-9bd92d20fc2d11e9bf63b974d90bef6c": [
+      "- {RoomControlBackend.CurrentState.Message}"
+    ],
+    "# OpenClose-RoomControlBackendonFailure-9bd92d20fc2d11e9bf63b974d90bef6c": [
+      "- Sorry, unable to connect to the room control service right now"
+    ],
+    "# TurnOnOff-RoomControlBackendonSuccess-7ca4cca000e411ea97e43706794592d8": [
+      "- {RoomControlBackend.CurrentState.Message}"

```

Comment:
Any idea why the "Sorry, unable to connect to the room control service right now." section is repeated 6 times here?